docs/tutorial/103-example.md

```diff
@@ -0,0 +1,108 @@
+| Text generation     | `openai`           | Standard OpenAI chat API, FastChat and vllm                     |
+| Image generation   | `openai_dall_e`    | DALL-E API for generating images                             |
+| Embedding | `openai_embedding` | API for text embeddings                                      |
+| General usages in POST       | `post_api`         | Huggingface/ModelScope Inference API, and customized post API |
+
+
+Each API has its specific configuration requirements. For example, to configure an OpenAI API, you would need to fill out the following fields in the model config in a dict, a yaml file or a json file:
+
+```
+model_config = {
+    "type": "openai",               # Choose from "openai", "openai_dall_e", or "openai_embedding"
+    "name": "{your_config_name}",   # A unique identifier for your config
+    "model_name": "{model_name}",   # The model identifier used in the OpenAI API, such as "gpt-3.5-turbo", "gpt-4", or "text-embedding-ada-002"
+    "api_key": "xxx",               # Your OpenAI API key. If unset, the environment variable OPENAI_API_KEY is used.
+    "organization": "xxx",          # Your OpenAI organization ID. If unset, the environment variable OPENAI_ORGANIZATION is used.
+}
+```
+
+For open-source models, we support integration with various model interfaces such as HuggingFace, ModelScope, FastChat, and vllm. You can find scripts on deploying these services in the `scripts` directory, and we defer the detailed instructions to [[Using Different Model Sources with Model API]](https://github.com/alibaba/AgentScope/tree/main/docs/tutorial/203-model.md).
+
+You can register your configuration by calling AgentScope's initilization method as follow. Besides, you can also load more than one config by calling init mutliple times.
+```python
+import agentscope
+
+agentscope.init(model_configs="./openai_model_configs.json")
+agentscope.init(model_configs="./modelscope_model_configs.json") # use openai and modelscope at the same time
+
+# or you can init once by passing a list of config dict
+# openai_cfg_dict = {...dict_filling...}
+# modelscope_cfg_dict = {...dict_filling...}
+# agentscope.init(model_configs=[openai_cfg_dict, modelscope_cfg_dict])
+```
+
+
+## Step2: Create Agents
+
+Creating agents is straightforward in AgentScope. After initializing AgentScope with your model configurations (Step 1 above), you can then define each agent with its corresponding role and specific model.
+
+```python
+import agentscope
+from agentscope.agents import DialogAgent, UserAgent
+
+# read model configs
+agentscope.init(model_configs="./openai_model_configs.json")
+
+# Create a dialog agent and a user agent
+dialogAgent = DialogAgent(name="assistant", model="gpt-4")
+userAgent = UserAgent()
+```
+
+**NOTE**: Please refer to [[Customizing Your Custom Agent with Agent Pool]](https://github.com/alibaba/AgentScope/tree/main/docs/tutorial/202-agent.md) for all available agents.
+
```

Comment:
the above hyperlink should be '202-agent' -> '201-agent' ？

Reply:
yes. fixed